dealership/contracts.py

```diff
@@ -4,9 +4,60 @@ class Contract(object):
 
 class BuyContract(Contract):
     def __init__(self, vehicle, customer, monthly_payments):
-        pass
-
-
-class LeaseContract(Contract):
+        self.vehicle = vehicle 
+        self.customer = customer 
+        self.monthly_payments = monthly_payments
+    
+    def discount_car(self, vehicle_price): 
+        return vehicle_price - vehicle_price * .1
+        
+    def total_value(self):
+        if self.vehicle.is_car() and self.customer.is_employee() == False:
```

Comment:
So simple how you used self.customer.is_employee for if the customer was a employee. Really didn't think about that